_std/macros/reagents.dm

```diff
@@ -0,0 +1 @@
+#define probmult(x) (prob(percentmult((x), mult))) //assumes lag-compensation multiplier is always called mult
```

Comment:
could take in the lag-compensation multiplier as an arg

Reply:
Unless I missed something very obvious optional arguments aren't a thing with macros. However, that can be fixed by clever use of a few macros + varargs probably.

Reply:
Probably could, but maybe not sanely - using the `percentmult(x, mult)` macro in a prob call is probably better practice anyways, but this was easier 🤷